lib/server/openapi.yaml

```diff
@@ -242,8 +277,33 @@ components:
         gitBranch:
           type: string
           description: Git branch used when cloning the repository. If not specified will use the default branch assigned to the repository
-
-
+        lifecycle:
+          type: string
+          description: Product lifecycle for the generated BOM. Choices are pre-build, build, post-build.
+        deep:
+          type: boolean
+          description: Perform deep searches for components. Useful while scanning C/C++ apps, live OS and oci images.
+          default: false
+        profile:
+          type: string
+          description: BOM profile to use for generation. Default generic. Choices are appsec, research.
```

Comment:
To support license-compliance, we need to move away from using environment variables.